### ISSUE TITLE
Login page: User role Dropdown is identified as a combobox

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4638]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2 Name, Role, Value and 2.1.1 keyboard

**Test Environment:**
- Test URL: https://clarity.design/documentation/login
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL
2. Move screen reader focus on User role dropdown and observe the spoken feedback.

**Expected result:**
Name, role and value should be appropriately announced for the focused element/dropdown.

**Actual result:**
Dropdown is identified as combobox. Traditionally, it is a combination of a drop-down list or list box and a single-line editable textbox. In the element in question, we don't have any editable textbox, so it cannot identified as a combobox.
As it's configured as combobox, keyboard only users cannot expand it using enter key. Only space and arrow keys are working.

**Comment by gkhadK** [comment]
Same issue is observed for the dropdown list box component provided on Select and Forms page.

**Comment by gkhadK** [comment]
Working as intended.
